life-and-letters/modernized/03-letters.adoc

```diff
@@ -8942,12 +9002,16 @@ to be thankful for the renewed mercies extended in a collective sense,
 for I think we may say that the meetings yesterday closed under a precious
 sense of the goodness and power of our blessed Master.
 
+[.signed-section-closing]
 With love to you and yours,
 
+[.signed-section-closing]
 I remain,
 
+[.signed-section-closing]
```

Comment:
same with this, one, i think we should merge all 3 of them into one closing. (and same for both of these in the other edition too)